package.json

```diff
@@ -36,6 +36,7 @@
     "graphql": "^0.6.0",
     "lodash": "^4.13.1",
     "newrelic": "^1.28.3",
+    "node-fetch": "^1.6.0",
```

Comment:
it's used in prismic-backup-service, but it mentions node-fetch in the package.json there.
Definitely remove from this package.json  -but just to bear in mind in case anything weirdly fails!